### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by zealott6** [Chia-Network/chia-blockchain#3168] [General]
LLevo 8 horas esperando desde que actualize el nuevo cliente y no me sincroniza.

**Comment by sargonas** [comment]
There are multiple things that can lead to a slow sync, and a little bit of trial and error trouble shooting will likely sort you. Tagging this issue for migration to a discussion where you will be best served by moving this over there and out of the issue tracker (assuming you are still experiencing this issue), where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
